pkgs/top-level/aliases.nix

```
@@ -408,6 +415,7 @@ mapAliases ({
   icedtea8_web = adoptopenjdk-icedtea-web; # added 2019-08-21
   icedtea_web = adoptopenjdk-icedtea-web; # added 2019-08-21
   idea = jetbrains; # added 2017-04-03
+  idrisPackages.protobuf = throw "idrisPackages.protobuf has been removed: abandoned by upstream"; # added 2022-02-06
```

This wont work and overrides idrisPackages to only contain protobuf. This need to be moved to where idrisPackages is declared.

```suggestion
  idrisPackages.protobuf = throw "idrisPackages.protobuf has been removed because abandoned by upstream"; # added 2022-02-06
```